keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -766,59 +848,67 @@ type ResourceHandle struct {
     // The maximum size of this field is 16KiB. This may get increased in the
     // future, but not reduced.
     // +optional
-    Data *string
+    Data string
```

Comment:
Was this just incorrect from before? I assume we didn't actually make this change just now in 1.30...

Reply:
The code continues to use a normal string. Changing to a pointer didn't provide any benefit when I worked on the implementation, so I didn't switch after all.